client/state/selectors/get-onboarding-url.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import { getVariationForUser } from 'calypso/state/experiments/selectors';
+import { getCurrentUserLocale } from 'calypso/state/current-user/selectors';
+import config from 'calypso/config';
+
+/**
+ * Returns the onboarding URL.
+ *
+ *
+ * @param {object}  state Global state tree
+ * @returns {string}  URL of the onboarding flow for existing users.
+ */
+
+export default function getOnboardingUrl( state ) {
+	const userLocale = getCurrentUserLocale( state );
+	if ( [ 'en', 'en-gb' ].includes( userLocale ) ) {
```

Comment:
Nit: can we define this array outside of the selector so it doesn't get redeclared on every selector call?